experimental/language-generation/csharp_dotnetcore/06.using-cards/Resources/Cards.LG

```diff
@@ -6,10 +6,42 @@
 
 # AllCards
 [Activity
-    Attachments = @{HeroCard()} | @{ThumbnailCard()} | @{SigninCard()} | @{AnimationCard()} | @{VideoCard()} | @{AudioCard()} | @{AdaptiveCard()}
+    Attachments = @{HeroCard()} | @{ThumbnailCard()} | @{SigninCard()} | @{AnimationCard()} | @{VideoCard()} | @{AudioCard()} | @{json(adaptivecardjson())}
```

Comment:
signin card here is 
```
# SigninCard
[SigninCard
    text = BotFramework Sign-in Card
    buttons = Sign-in
]
```

The button is just a normal message back button (default), may we change it to a 'signin' button in this sample? like:

```
# SigninCard
[SigninCard
    text = BotFramework Sign-in Card
    buttons = @{signinButton()}
]

# signinButton
[CardAction
 title =  Sign-in
 type = signin
 value = http://login.microsoft.com
]
```